src/itemdef.cpp

```diff
@@ -175,7 +173,7 @@ void ItemDefinition::deSerialize(std::istream &is)
 
 	// Deserialize
 	int version = readU8(is);
-	if (version < 1 || version > 4)
+	if (version < 5 || version > 5)
```

Comment:
`version != 5`

Reply:
```
-	u8 version = (protocol_version >= 34) ? 4 : 3;
+	u8 version = 5;
```
New version is 5, previous was 4. Maybe because the old code had `|| version > 4`?

Reply:
I know that, but why would you explicitly refuse to support likely backwards-compatible <i>future</i> versions?

Reply:
Exactly, it's weird.